WordPress/Classes/Services/PeopleService.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+
+class PeopleService {
+    let remote: PeopleRemote
+    let siteID: Int
+
+    private let context = ContextManager.sharedInstance().mainContext
+
+    init(blog: Blog) {
+        remote = PeopleRemote(api: blog.restApi())
+        siteID = blog.dotComID() as Int
+    }
+
+    func refreshTeam(completion: (Bool) -> Void) {
+        remote.getTeamFor(siteID,
+            success: {
+                (people) -> () in
+
+                self.mergeTeam(people)
```

Comment:
I'm not sure about this one. If we're at the point where we're ready to call the success block, I really want to call `mergeTeam` so we don't waste the downloaded data.

When I run Instruments I don't see any leaks or retain cycles